modules/navigation/register_types.cpp

```diff
@@ -32,20 +32,19 @@
 
 #include "godot_navigation_server.h"
 
-#ifndef _3D_DISABLED
-#include "navigation_mesh_generator.h"
-#endif
+#include "core/config/engine.h"
 
 #ifdef TOOLS_ENABLED
 #include "editor/navigation_mesh_editor_plugin.h"
 #endif
 
-#include "core/config/engine.h"
-#include "servers/navigation_server_3d.h"
-
+#ifndef DISABLE_DEPRECATED
 #ifndef _3D_DISABLED
+#include "navigation_mesh_generator.h"
+
 NavigationMeshGenerator *_nav_mesh_generator = nullptr;
 #endif
+#endif // DISABLE_DEPRECATED
```

Comment:
The previous include order was correct, see #75944.

It should be the "local" (`modules/navigation/` relative) includes first, then the "global" (`core/`, `servers/`, etc.) includes.

Here it makes a case which is slightly convoluted and neither convention is better than the other, but it's good to stay consistent with the rest of the codebase.